Core/NoOpRunner.Core/PlayerStates/IdleState.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace NoOpRunner.Core.PlayerStates
 {
-    class IdleState : PlayerState
+    public class IdleState : PlayerState
```

Comment:
Do not change the classes to public - instead, use the `InternalsVisibleTo` [attribute](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.internalsvisibletoattribute?redirectedfrom=MSDN&view=net-5.0) so that the test assembly can see the internal classes of the required assemblies. Does not matter too much in our case, as the class visibility is all over the place anyways, but it's a good practice and it's more convenient.

Reply:
Fixed